src/main/java/org/broadinstitute/hellbender/tools/walkers/longreads/CountIndels.java

```diff
@@ -0,0 +1,282 @@
+package org.broadinstitute.hellbender.tools.walkers.longreads;
+
+import com.google.common.base.Joiner;
+
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.LongReadProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Quickly count errors
+ *
+ * <h3>Input</h3>
+ * <ul>
+ *     <li>A single BAM file</li>
+ * </ul>
+ *
+ * <h3>Outputs</h3>
+ * <ul>
+ *     <li>A collection of BAM files where care has been keep reads from the same ZMW in the same file</li>
+ * </ul>
+ *
+ * <h3>Usage Example</h3>
+ * <h4>Quickly count errors</h4>
+ * <pre>
+ *   gatk CountIndels \
+ *     -I input.bam \
+ *     -L region.bed \
+ *     -O output.txt
+ * </pre>
+ */
+@DocumentedFeature
+@ExperimentalFeature
+@CommandLineProgramProperties(
+        summary = "Count indels",
+        oneLineSummary = "Count indels",
+        programGroup = LongReadProgramGroup.class
```

Comment:
This is one of the tools in here that probably can belong into one of the other Program groups if you included much more detailed documentation about what it does.